script.js

```diff
@@ -3,6 +3,28 @@ const menuList = document.querySelector('.navlinks');
 const menuItems = Array.from(document.querySelectorAll('.navlink-items'));
 const menuIcon = document.querySelector('.menu-icons');
 const body = document.querySelector('.page-container');
+const pB = document.querySelector('.modal-body');
+const pB2 = document.querySelector('.modal-body-desktop');
+const projectCards = document.querySelector('#Works-grid');
+
+const mobilePopUp = {
+  Card_title: 'Keeping track of hundreds of components',
+  project_img: './style/images/Disabled.svg',
+  x_image: './style/images/Snapshoot-Portfolio.svg',
+  button_image1: './style/images/P-github-icon.svg',
+  button_image2: './style/images/see-live-icon.svg',
+  project_description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  tags: ['Ruby on rails', 'css', 'javascript'],
+  button_name1: 'See Live',
+  button_name2: 'See Source',
+  live_version: 'https://hamid-murambiwa.github.io/Portfolio/',
+  source_code: 'https://github.com/hamid-murambiwa/Portfolio',
+};
```

Comment:
- [ ] <b>REQUIRED:</b> You don't need this. just make sure that every data you need about each project is included in each item of the projects list array.